x-pack/test_utils/enzyme_helpers.js

```diff
@@ -4,42 +4,86 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
```

Comment:
__suggestion:__ it's unfortunate that we should duplicate enzyme_helpers in both oss and x-pack. Can you please file an issue with `i18n`, `test` and `Platform` labels to get rid of this duplication at some point. It will require a couple of steps:

1. Define `test_utils` alias in Jest x-pack config (like we have in OSS Jest config);
2. Somehow use `enzyme` from x-pack when this file is used in x-pack, and use `enzyme` from oss when it's used from oss. Enzyme uses global configuration under the hood, so the `enzyme\adapter` you configure at x-pack Jest setup step isn't the same as one required by `enzyme_helpers` in OSS so all methods `mount/shallow/render` will complain that `enzyme` isn't configured if you try to use `enzyme_helpers` from oss in x-pack tests.

Reply:
Sure, will create an issue soon